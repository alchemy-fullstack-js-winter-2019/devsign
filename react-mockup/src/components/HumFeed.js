import React from 'react';
import Header from './Header';
import HumForm from './HumForm';
import Footer from './Footer';
import Hums from './Hums';

function HumFeed() {
  return (
    <>
    <Header />
    <main>
      <section>
        <h2>Your feed</h2>
        <HumForm />
        <Hums />
      </section>
    </main>
    <Footer />
  </>
  );
}

export default HumFeed;
