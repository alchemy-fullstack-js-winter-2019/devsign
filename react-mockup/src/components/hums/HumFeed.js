import React from 'react';
import Header from '../app/Header';
import HumForm from './HumForm';
import Footer from '../app/Footer';
import Hums from './Hums';
import styled from 'styled-components';

const Heading = styled.h2`
  font-family: 'Montserrat', sans-serif;
  text-align: center; 
  border-top: 1px solid #E0E0E0;
  padding-top: 25px;
`;

function HumFeed() {
  return (
    <>
    <Header />
    <main>
      <section>
        <Heading>hum-feed</Heading>
        <HumForm />
        <Hums />
      </section>
    </main>
    <Footer />
  </>
  );
}

export default HumFeed;
