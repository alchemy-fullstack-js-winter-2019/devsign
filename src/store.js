import { createStore } from 'redux';
import reducer from './reducers/tweets';

const store = createStore(
  reducer,
  window.__REDUX_DEVTOOLS_EXTENSION__ && window.__REDUX_DEVTOOLS_EXTENSION__()
);

export default store;
