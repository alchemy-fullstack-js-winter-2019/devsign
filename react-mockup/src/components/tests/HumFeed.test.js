import React from 'react';
import HumFeed from '../hums/HumFeed';
import { MemoryRouter } from 'react-router';
import { Provider } from 'react-redux';
import store from '../../store';
import { shallow } from 'enzyme';
import Enzyme from 'enzyme';
import Adapter from 'enzyme-adapter-react-16';

Enzyme.configure({ adapter: new Adapter() });

jest.mock('../../services/auth.js');

describe('Humfeed Snapshot', () => {
  it('can match a snapshot', () => {
    const tree = shallow(
      <Provider store={store}>
        <MemoryRouter>
          <HumFeed />
        </MemoryRouter>
      </Provider>
    );
    expect(tree).toMatchSnapshot();
  });
});
