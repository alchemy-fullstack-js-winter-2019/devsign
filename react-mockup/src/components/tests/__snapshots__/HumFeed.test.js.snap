// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Humfeed Snapshot can match a snapshot 1`] = `ShallowWrapper {}`;
